docs/how-it-works/features/security/overview.md

```diff
@@ -0,0 +1,19 @@
+![binbash-logo](../../../assets/images/logos/binbash.png "Binbash"){: style="width:250px"}
+![binbash-leverage-tf](../../../assets/images/logos/binbash-leverage-terraform.png#right "Leverage"){: style="width:130px"}
+
+# AWS Reference Architecture security features
+
+??? check "VPN"
+    - [x] [VPN](./vpn.md)
+
+??? check "Services"
+    - [x] [Services](./services.md)
+
+??? check "Certificates"
+    - [x] [Certificates](./certificates.md)
+
+??? check "Audit | CloudTrail"
+    - [x] [CloudTrail](./audit-cloudtrail.md)
+
+??? check "IAM Access Analyzer"
```

Comment:
@eze-godoy In the overview section I think we should refer to the high level sections so here we'll only present "Monitoring" and then add all the Security Monitoring related services as child elements. For example we could add "GuardDuty" in the future, and we expect it to be added as follows:

```
??? check "Monitoring"
        - [x] [IAM Access Analyzer](./iam-access-analyzer.md)
        - [x] [GuardDuty](./monitoring-guardduty.md)
```